cli.js

```diff
@@ -67,6 +67,8 @@ lighthouse({
   } else {
     prettyPrint(results);
   }
+}).catch(function () {
+  process.exit();
```

Comment:
I was digging it more and I found that it takes some time over 9sec to exit process, It's not hanging of process. I'm not sure about that it's related to my env.